.node-version

```diff
@@ -0,0 +1 @@
+14.17.0
```

Comment:
Current LTS is v18.17.0, can we update this?

Reply:
What do you think about opening another task to update node version at the same time also on ecommerce-tests ?